packages/expo-task-manager/CHANGELOG.md

```diff
@@ -6,4 +6,6 @@
 
 ### 🎉 New features
 
+- Upgraging and application does not cause BackgroundFetch tasks to unregister. ([#8348](https://github.com/expo/expo/pull/8438) by [@mczernek](https://github.com/mczernek))
```

Comment:
```suggestion
- Upgrading an application does not cause `BackgroundFetch` tasks to unregister. ([#8348](https://github.com/expo/expo/pull/8438) by [@mczernek](https://github.com/mczernek))
```

Also, I think it's more a fix than a new feature 😄 
Could you check whether there are any reported issues related to background fetch that this PR may solve and link them in this PR to close them automatically?